pfsimulator/parflow_lib/solver_richards.c

```diff
@@ -1214,7 +1214,10 @@ SetupRichards(PFModule * this_module)
     /*-------------------------------------------------------------------
      * Print out the initial well data?
      *-------------------------------------------------------------------*/
-
+    WriteReservoirs("ReservoirsOutput",
```

Comment:
Add a key to control writing the output, like the print_wells flag.   We allow users to turn off I/O if they don't want it.